transports/sql/connection-settings_connection-string_sqltransport_[4,).partial.md

```diff
@@ -0,0 +1,12 @@
+### Via the configuration API
```

Comment:
u cant use a core snippet in a doco page that targets the sql transport component